core/services/periodicbackup/backup.go

```diff
@@ -0,0 +1,139 @@
+package periodicbackup
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/smartcontractkit/chainlink/core/logger"
+)
+
+var (
+	fileName           = "db_backup.tar.gz"
```

Comment:
Can we call it `cl_backup`, add a timestamp and add it to the gitignore?

Reply:
yeah